client/src/components/ProductOverview/ProductOverview.jsx

```diff
@@ -0,0 +1,113 @@
+import React from 'react';
+import axios from 'axios';
+
+import ImageGallery from './ImageGallery';
+import ProductInformation from './ProductInformation';
+import StyleSelector from './StyleSelector';
+import AddToCart from './AddToCart';
+import OverviewAndShare from './OverviewAndShare';
+
+class ProductOverview extends React.Component {
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isFetching: true,
+      expanded: false,
+      product: {},
+      productStyles: {},
+      productReviews: {},
+      productRatings: {},
+      productStarRatings: 3.7, // Hardcoded for now to play around with it
+    };
+  }
+
+  componentDidMount() {
+    Promise.all([
+      // For now, grab all products from the API and set the current product as the 1st one (40344)
+      ProductOverview.getProductList(),
+      // For now, grab product details only for the 1st product
+      ProductOverview.getProductStyles(40344),
```

Comment:
minor - can use this.getProductList() instead of ProductOverview.getProductList()

Reply:
see below - but when you use "static" methods you can't use "this" to refer to them anymore; you have to use the name of the class